src/types.rs

```diff
@@ -289,6 +289,19 @@ where
 {
     debug!("format_function_type {:#?}", shape);
 
+    let ty_shape = match context.config.indent_style() {
+        // 4 = " -> "
+        IndentStyle::Block => shape.offset_left(4)?,
+        IndentStyle::Visual => shape.block_left(4)?,
+    };
+    let output = match *output {
```

Comment:
Is the `output` variable the return type ? If so, maybe a more explicit variable name would be nice, e.g., `return_type` or something like that.

Reply:
sorry didn't check the function it was in, but in together with the function's name (`format_function_type`), then that's probably fine...